mk-files/release-test.mk

```diff
@@ -18,13 +18,13 @@ verify-archive-installer:
 # if ARCHIVES_VERSION is empty, latest folder will be tested
 .PHONY: test-installer
 test-installer:
-	@echo Running packaging/installer tests
-	@bash test-installer.sh $(ARCHIVES_VERSION)
+	$(call dry-run,bash test-installer.sh $(ARCHIVES_VERSION))
```

Comment:
How is this done not downloading from prod/stag?

Reply:
Not sure what you mean, but I chose to gate this with the `DRY_RUN` flag because since nothing has been uploaded, we can't test downloading.

Reply:
This is something that we should wait to do a release to test. We can't really test it without first uploading the new binary.

Of course, we could always test it using the old binary, but that seems counterintuitive to me...